GoogleForCreators/web-stories-wp
# Allow changing hex value of multi-selection mixed color

## Summary
Adjusts the color picker so that in case of mixed values, hex input would be editable as in case of other inputs.


## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions


I
1. Add two shape elements
2. Assign different colors to both
3. Select both elements.
4. Verify that clicking on "Mixed" does not open the color picker, instead, it can be directly edited in the input.
5. Verify the same when assigning different gradients to the two shapes.

II
1. Add one shape with a gradient
2. Verify that in this case, the color picker works as before -- in case of a gradient (e.g. "Linear"), clicking on the gradient text the color picker opens instead of being able to modify it directly. In case of solid color and hex value, the value can be changed directly from the input though.

---



Fixes #2809
